hack/build.sh

```diff
@@ -49,6 +49,9 @@ dirs=${VERSION:-$VERSIONS}
 
 for dir in ${dirs}; do
   IMAGE_NAME="${BASE_IMAGE_NAME}-${dir//./}-${OS}"
+  if [ "$dir" == "2.6" ]; then
+    IMAGE_NAME=rhscl/${BASE_DIR_NAME}-${dir//./}-${OS}  
```

Comment:
@rhcarvalho this is ultimately where the rhcl/mongodb-26-centos7 image is going to be coming from..so it's not there today, but it will be there once this gets merged and we build/push it.

however I don't think our push job is currently setup to handle this sort of naming, so that's going to require some changes if we're going to use the existing push job to push to dockerhub under an rhcl namespace instead of openshift.  i'll deal with that when i'm updating the push job to handle mongo 2.6 after this merges, though.